src/display/editor/freetext.js

```diff
@@ -370,6 +354,21 @@ class FreeTextEditor extends AnnotationEditor {
     return this.div;
   }
 
+  /** @inheritdoc */
+  static deserialize(data, parent) {
+    const editor = super.deserialize(data, parent);
+
+    editor.#fontSize = data.fontSize;
+    editor.color = Util.makeHexColor(...data.color);
```

Comment:
Is this correct, since I'd expect the following?
```suggestion
    editor.#color = Util.makeHexColor(...data.color);
```